cumulusci/cli/config.py

```diff
@@ -67,18 +67,25 @@ def alert(self, message="We need your attention!"):
             return
         click.echo("\a")
         try:
-            call(
-                [
-                    "osascript",
-                    "-e",
-                    'display notification "{}" with title "{}"'.format(
-                        message.replace('"', r"\"").replace("'", r"\'"), "CumulusCI"
-                    ),
-                ]
-            )
+            call(self._get_platform_alert_cmd(message))
         except OSError:
             pass  # we don't have osascript, probably.
 
+    def _get_platform_alert_cmd(self, message):
+        message = message.replace('"', r"\"").replace("'", r"\'")
```

Comment:
I think we only want this escaping for interpolation into the osascript statement, not for notify-send where it's passed verbatim as an arg (since we're using subprocess.call without shell=True)